lib/resources/users.js

```diff
@@ -113,7 +113,8 @@ module.exports = (service, endpoint) => {
     auth.actor.map((actor) =>
       ((queryOptions.extended === true)
         ? Promise.all([ Users.getByActorId(actor.id).then(getOrNotFound), Auth.verbsOn(actor.id, '*') ])
-          .then(([ user, verbs ]) => Object.assign({ verbs }, user.forApi()))
+          .then(([ user, verbs ]) => Users.getPreferences(user.actorId)
```

Comment:
I noticed `getPreferences` was added to the Users query module, copied over from the UserPreferences query module. Why not just change the endpoint to `... endpoint(({ Auth, Users, UserPreferences }...` and use `UserPreferences.getForUser(user.actorId)` directly?